Added txpool_pantheonStatistics (#1354)
PegaSysEng/pantheon: docs/Reference/JSON-RPC-API-Methods.md

    The `TXPOOL` API methods are not enabled by default. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
    or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `TXPOOL` API methods.

### txpool_pantheonStatistics

Lists statistics about the node transaction pool. 

**Parameters** 

None

**Returns** 

`result` - Transaction pool statistics: 

* `maxSize` - Maximum number of transactions kept in the transaction pool. Use the [`--tx-pool-max-size`](Pantheon-CLI-Syntax.md#tx-pool-max-size)
 option to configure the maximum size. 

> * `localCount` - Number of transactions submitted from the local node 

Reviewer: ```suggestion
* `localCount` - Number of transactions submitted directly to this node 
```
